sympy/functions/special/tests/test_bessel.py

```diff
@@ -273,6 +273,12 @@ def test_jn_zeros():
     assert eq(jn_zeros(3, 4), [6.987932, 10.417118, 13.698023, 16.923621])
     assert eq(jn_zeros(4, 4), [8.182561, 11.704907, 15.039664, 18.301255])
 
+    assert eq(jn_zeros(0, 4, method='scipy'), [3.141592, 6.283185, 9.424777, 12.566370])
```

Comment:
scipy is not a requirement for SymPy, so this will (and should) fail. Plus that it is already tested elsewhere. I'd say revert bessel as per my previous comment. 

Using a different approach than the user asks for is not better than throwing an exception.

Reply:
I have modified jn_zeros if no scipy, it we switch to sympy.

Reply:
see line 945 of bessel.py

Reply:
Yes, I will revert my code in bessel